src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
+  if (operation === '%') {
+    return one.mod(two).toString();
+  }
```

Comment:
- [ ] Your app crashes when I try to perform a modulo operation using 0. This is because you are not using the correct `operate.js` file provided in the project guidelines. Kindly use the correct `operate.js` [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js).

  ![vseray-modulo](https://user-images.githubusercontent.com/36542986/153137188-22d93c39-0300-40bf-8d3e-99108ad5d61c.jpg)